modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -1,26 +1,76 @@
+
+The agent also listens for incoming requests from the control plane. When the agent receives a message from the control plane, the agent makes changes to the appropriate Kubernetes resources as specified in the message. Such changes include creating a new application, updating an existing application, or deleting an application. 
+
+=== Additional Cluster Components
+
+The following Runtime Fabric Components also run inside the Kubernetes cluster: 
+
+* Mule Cluster IP Service: Provides an API that Mule applications use to discover their peers inside the cluster
+* Resource cache: Provides a cluster-local cache of application dependencies
+* Persistence Gateway: Provides a persistent ObjectStore interface to Mule applications
+
+=== About Component Installations and Upgrades
+
+When you install Runtime Fabric on your Kubernetes cluster, the Runtime Fabric command-line utlity (`rtfctl`) swaps an activation data token for a set of activation properties. Properties include credentials to access the Runtime Fabric Docker Registry and credentials to establish an mTLS connection to the Anypoint platform. It then invokes Helm to perform the installation. 
+
+You trigger upgrades to the Runtime Fabric components in your cluster from the Anypoint control plane, specifically from Runtime Manager. When you trigger an upgrade, an upgrade message is processed inside the target cluster, and `rtfctl` invokes Helm to upgrade the components' software versions. If you encounter a problem during an upgrade, you can use `rtfctl` to invoke an Helm rollback and restore the cluster to a stable state. A rollback creates a report in the control plane.
+
+== How Application Deployments Work in Runtime Fabric on Self-Managed Kubernetes 
+
+When you deploy an application in Runtime Fabric on Self-Managed Kubernetes, the following occurs:
+
+. You use Runtime Manager to trigger the application deployment.
+. The Runtime Fabric agent in the target cluster receives a request to deploy the application.
+. Runtime Fabric assigns the application's namespace or creates a new namespace if necessary.
+. Runtime Fabric generates the appropriate Kubernetes resources, including a deployment, ConfigMap, secrets, services, and ingress.
+. The Kubernetes deployment resource may include an init container, which fetches dependencies from the Runtime Fabric resource cache.
+. If the resource cache doesn’t contain required dependencies, Runtime Fabric fetches them from the control plane and adds them to the resource cache.
+
+//image 
+
+=== Assigning Namespaces in Runtime Fabric
+
+Each application is deployed into a Kubernetes namespace based on the application’s environment. 
+
+. Runtime Fabric looks for a namespace with the label `rtf.mulesoft.com/envId=<ANYPOINT_ENVIRONMENT_ID>`. 
+. If Runtime Fabric can't find that label, it looks for a namespace with the name `<ANYPOINT_ENVIRONMENT_ID>`. 
```

Comment:
```suggestion
. If Runtime Fabric can't find that label, it searches for a namespace with the name `<ANYPOINT_ENVIRONMENT_ID>`. 
```

Reply:
Is this ^^^ a concatenated representation of the previous full search string, indicating a repetition of the search, just with a different name (EKS, AKS, or GKS)?